percona/extensions/uninstall.go

```diff
@@ -42,3 +47,28 @@ func Uninstall(archivePath string) error {
 
 	return nil
 }
+
+func DisableCustomExtensionsInPostgreSQL(ctx context.Context, customExtensionsForDeletion []string, exec postgres.Executor) error {
```

Comment:
I would name it simply like `DropExtension`.

Reply:
@inelpandzic We have similar method for pgaudit and I decided to keep the same name for consistency. What do you think? 
But I agree that the name you suggested is easier.